portfolio/src/main/webapp/index.html

```diff
@@ -51,18 +51,19 @@ <h1>Contact Me:</h1>
           </tr>
       </table>
 
-      <form action="/form-handler" method="POST">
+      <form action="/form-handler" method="POST" style="margin-top: 10px; margin-bottom: 10px;">
         <p>Enter any comments or suggestions below!</p>
         <!-- TODO: Will eventually add ability to input title comment -->
-        <!-- <input name="title" placeholder="Enter a title"> <br/> -->
-        <!-- <textarea name="comment-suggestion" placeholder="Enter Comment/Suggestion"></textarea> -->
+        <input name="title" placeholder="Enter a title"> <br/>
+        <textarea name="comment-suggestion" placeholder="Enter Comment/Suggestion"></textarea>
+        <!-- <input name="comment-suggestion" placeholder="Enter Comment/Suggestion"> -->
 
-        <input name="comment-suggestion" placeholder="Enter Comment/Suggestion">
         <br/>
         <button type="submit">Submit</button>
       </form>
 
-      <a href="comments.html">View other comments here!</a>
+      <a href="comments.html" style="margin-bottom: 30px;">View other comments here!</a>
```

Comment:
Optional: same css class comment here as above

Reply:
Same as above, I figured I put the styling within the HTML since it wasn't a lot. I'll try to extract styling to CSS in the group project!